ephemeraldata.adoc

```diff
@@ -0,0 +1,59 @@
+= Ephemeral Data
+
+There are 3 main types of ephemeral data stored in LDAP during normal operation of the {product-name}.
+
+      - Last Logon Time Stamps
+      - Auth Tokens
+      - CSRF Tokens
+
+Historically the only option for storage of these types of ephemeral data was in LDAP.
+Starting with version 8.8.0 the ability to save these types of ephemeral data using an external server has been implemented.  Note: This document does not cover how to install and maintain the ephemeral storage server.
+
+Configuring the storage location of ephemeral data is done through the following LDAP attribute:
+
+|====================
+| Attribute | Format | Description
+| zimbraEphemeralBackendURL | [backend name]:[params] | Ephemeral Backend URL Configuration
+|====================
+
+The two currently supported Ephemeral Data backends are:
+
+|====================
+| Backend | Format | Description
+| LDAP    | ldap://default |  Default configuration
+| SSDB    | ssdb:127.0.0.1:8888 | SSDB server and port
+|====================
+
+Mail systems with large numbers of active users have been found to overload LDAP as noted in  https://bugzilla.zimbra.com/show_bug.cgi?id=104858[Bugzilla 104858].
+
+== Post {product-name} Install Configuration
+
+Configuring an already running {product-name} installation
+to utilize `SSDB` instead of `LDAP` for short lived data storage
+through the following process:
+1. Install `SSDB` and note the ip address and port configured since you will
+   need this data for the next steps.
+2. Configure the {product-name} to utilize `SSDB`.
+3. Migrate any existing short-lived data to `SSDB` using the `/opt/zimbra/bin/zmmigrateattrs` command.
+
+==== Migration Procedure
+
+1. Access the command prompt on one of the machines in the installation.
+2. Configure the {product-name} to use `SSDB`:
+
```

Comment:
There is a cleaner way to have multiple things in a single list entry using "Open Blocks" for list item continuation
See http://asciidoc.org/userguide.html#X29
For an example of similar changes I made for **installguides** - see 
`git show 2763fdca5048982b1d39c4e2eff6f15c141173fd`